processor/src/bitwise/mod.rs

```diff
@@ -263,14 +267,14 @@ impl Bitwise {
     // HELPER METHODS
     // --------------------------------------------------------------------------------------------
 
-    /// Appends a new row to the trace table and populates the first 12 column of trace as follows:
+    /// Appends a new row to the trace table and populates the first 13 columns of trace as follows:
```

Comment:
nit: not a big deal, but maybe this should say "15 columns" instead of 13. I'm not sure if the 12 before was an error from not being updated or if it was not counting the selector columns, but I think maybe it was also unclear. I'll leave it up to you to do whatever you think is the clearest option.

Reply:
I, too, had this confusion when I was working on it. I concluded that this "12 columns" doesn't include the selectors columns and thus updated it to 13 earlier. I agree with you and to make it consistent everywhere and I have updated it to 15 in the latest repo.